atd-etl/cris_import/cris_import.py

```diff
@@ -81,7 +82,21 @@ def main():
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
     # 🥩 & 🥔
-    zip_location = download_extract_archives()
+    #zip_location = download_extract_archives()
```

Comment:
this line should be removed, right? and as Mike pointed out, download_extract_archives and download_archives overlap, and this is the only place I could find that referenced download_extract_archives, so I'm guessing thats the one that can be deleted?